src/Emogrifier/HtmlProcessor/AbstractHtmlProcessor.php

```diff
@@ -146,7 +169,9 @@ private function createRawDomDocument($html)
      */
     private function prepareHtmlForDomConversion($html)
     {
-        $htmlWithDocumentType = $this->ensureDocumentType($html);
+        $htmlWithSelfClosingSlashes = $this->ensurePhpUnrecognizedSelfClosingTagsAreXml($html);
+
```

Comment:
Why the blank line?

Reply:
Because there was previously a blank line between the other two statements.  But I guess that is because the latter is the final `return` statement?  I'll remove the blank line.

Reply:
Yes, that was the reason.